rootfs/ingest/bin/env.sh

```diff
@@ -1,3 +1,10 @@
+# Parameters for ingest scripts
+
+data_url="$DATA_URL"
+# Strip trailing slash
+data_url=$(echo $data_url | sed 's%\(.*[^/]\)/*%\1%')
+
 PORT="25080"
-REPL_URL="http://qserv-repl-ctl-0.qserv-repl-ctl:${PORT}"
+QUEUE_URL="mysql://qsingest:@qserv-ingest-db-0.qserv-ingest-db:3306/qservIngest"
```

Comment:
Does it have to be the fixed port? And what about the port definition one lobe above? Is it still needed?
```
PORT="25080"
```

Reply:
Yes it is needed, but I'll remove it when integrating qserv-ingest inside qserv-operator, which is aware of the port number.